common/components/timeline/template.njk

```diff
@@ -24,6 +24,9 @@
             {{- item.datetime.timestamp | date(item.datetime.format) -}}
           {% endif -%}
         </time>
+        {%- if item.createdBy.text %}
```

Comment:
This already exists, on line 32 as `byline` :)

Reply:
:faceplant: I did not realise that's what byLine is.... Thank you!

Reply:
Done!